articles/data-factory/quickstart-create-data-factory-azure-cli.md

```diff
@@ -121,20 +121,19 @@ Next, create a linked service and two datasets.
 
    ```json
    {
-       "type": 
-           "AzureBlob",
-           "linkedServiceName": {
-               "type":"LinkedServiceReference",
-               "referenceName":"AzureStorageLinkedService"
-               },
-           "annotations": [],
-           "type": "Binary",
-           "typeProperties": {
-               "location": {
-                   "type": "AzureBlobStorageLocation",
-                   "fileName": "emp.txt",
-                   "folderPath": "input",
-                   "container": "adftutorial"
+       "type": "AzureBlob",
+       "linkedServiceName": {
+           "type": "LinkedServiceReference",
+           "referenceName": "AzureStorageLinkedService"
+       },
+       "annotations": [],
+       "type": "Binary",
```

Comment:
VS Code reports error for `type`. Please confirm if this is expected:

![image](https://user-images.githubusercontent.com/4003950/129521558-3890b928-893f-40f1-b81d-ee4b9880c634.png)